src/Egret.Cli/Serialization/Audacity/AudacitySerializer.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace Egret.Cli.Serialization.Audacity
+{
+    using Models;
+    using System.IO;
+    using System.IO.Abstractions;
+    using System.Threading.Tasks;
+    using Xml;
+
+    public partial class AudacitySerializer
+    {
+        private readonly DefaultXmlSerializer serializer;
+
+        public AudacitySerializer(DefaultXmlSerializer serializer)
+        {
+            this.serializer = serializer;
+        }
+
+        public async Task<Project> Deserialize(IFileInfo fileInfo)
+        {
+            var result = await serializer.Deserialize<Project>(fileInfo);
+            result.SourceInfo = new SourceInfo(fileInfo.FullName);
+            return result;
+        }
+
+        public Project Deserialize(Stream stream, string path)
+        {
+            var result = serializer.Deserialize<Project>(stream);
```

Comment:
clarify whether this can be `null`. I don't think it should be (if the file is empty it should be an empty `Project` instance